<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>TestNG Report</title><style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}</style></head><body><table><tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr><tr><th colspan="7">Default suite</th></tr><tr><td><a href="#t0">Default test</a></td><td class="num">5</td><td class="num">0</td><td class="num attn">5</td><td class="num">5,263</td><td></td><td></td></tr></table><table><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Default suite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">Default test &#8212; failed</th></tr><tr class="failedeven"><td rowspan="5">com.sapphire.etl.AERSDemoVerify</td><td><a href="#m0">demoCaseAgeCode</a></td><td rowspan="1">1401432567529</td><td rowspan="1">36</td></tr><tr class="failedeven"><td><a href="#m1">demoCaseAgeGroup</a></td><td rowspan="1">1401432567566</td><td rowspan="1">4345</td></tr><tr class="failedeven"><td><a href="#m2">demoCaseGenderCode</a></td><td rowspan="1">1401432571913</td><td rowspan="1">36</td></tr><tr class="failedeven"><td><a href="#m3">demoCaseGenderDEC</a></td><td rowspan="1">1401432571950</td><td rowspan="1">41</td></tr><tr class="failedeven"><td><a href="#m4">demoCaseValidTo</a></td><td rowspan="1">1401432572060</td><td rowspan="1">33</td></tr><tr><th colspan="4">Default test &#8212; passed</th></tr><tr class="passedeven"><td rowspan="5">com.sapphire.etl.AERSDemoVerify</td><td><a href="#m5">demoCASE_ID</a></td><td rowspan="1">1401432566896</td><td rowspan="1">632</td></tr><tr class="passedeven"><td><a href="#m6">demoCaseSeriousness</a></td><td rowspan="1">1401432571992</td><td rowspan="1">29</td></tr><tr class="passedeven"><td><a href="#m7">demoCaseValidFrom</a></td><td rowspan="1">1401432572022</td><td rowspan="1">36</td></tr><tr class="passedeven"><td><a href="#m8">demoFDA_date</a></td><td rowspan="1">1401432572093</td><td rowspan="1">28</td></tr><tr class="passedeven"><td><a href="#m9">demoISR_ID</a></td><td rowspan="1">1401432572122</td><td rowspan="1">28</td></tr></tbody></table><h2>Default test</h2><h3 id="m0">com.sapphire.etl.AERSDemoVerify#demoCaseAgeCode</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT distinct age_code FROM aers_demo<br/>Expected : [YR, MON, WK, DY, HR, DEC, MIN, SEC, &apos;&apos;]<br/>Actual : [YR,  , MON, WK, DY, HR, DEC, M, F, 64, SEC, MIN, 85]</div></td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: SELECT distinct age_code FROM aers_demo
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERSDemoVerify.demoCaseAgeCode(AERSDemoVerify.java:352)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: null: lists don&apos;t have the same size expected [9] but found [13]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:537)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.sapphire.etl.AERSDemoVerify.demoCaseAgeCode(AERSDemoVerify.java:345)
	... 24 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m1">com.sapphire.etl.AERSDemoVerify#demoCaseAgeGroup</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where std_age_group = &apos;Not Available&apos; AND age != &apos;&apos; AND age_code != &apos;&apos; AND age not like (&apos;%-%&apos;) AND age not like (&apos;%.%&apos;) AND age != 0 AND age != 00<br/>Expected : [0]<br/>Actual : [631]</div></td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: SELECT count(*) FROM aers_demo where std_age_group = &apos;Not Available&apos; AND age != &apos;&apos; AND age_code != &apos;&apos; AND age not like (&apos;%-%&apos;) AND age not like (&apos;%.%&apos;) AND age != 0 AND age != 00
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERSDemoVerify.demoCaseAgeGroup(AERSDemoVerify.java:389)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: Lists differ at element [0]: 0 != 631 expected [0] but found [631]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:549)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.sapphire.etl.AERSDemoVerify.demoCaseAgeGroup(AERSDemoVerify.java:382)
	... 24 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m2">com.sapphire.etl.AERSDemoVerify#demoCaseGenderCode</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT distinct gender_code FROM aers_demo<br/>Expected : [F, M, UNK, NS, U, &apos;&apos;]<br/>Actual : [F,  , M, UNK, NS, null, YR, N, Y, U]</div></td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: SELECT distinct gender_code FROM aers_demo
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERSDemoVerify.demoCaseGenderCode(AERSDemoVerify.java:234)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: null: lists don&apos;t have the same size expected [6] but found [10]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:537)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.sapphire.etl.AERSDemoVerify.demoCaseGenderCode(AERSDemoVerify.java:227)
	... 24 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m3">com.sapphire.etl.AERSDemoVerify#demoCaseGenderDEC</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT distinct gender_dec FROM aers_demo<br/>Expected : null<br/>Actual : []</div></td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: SELECT distinct gender_dec FROM aers_demo
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERSDemoVerify.demoCaseGenderDEC(AERSDemoVerify.java:273)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.sql.SQLException: Column Index out of range, 2 &gt; 1. 
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1073)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:987)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:982)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:927)
	at com.mysql.jdbc.ResultSetImpl.checkColumnBounds(ResultSetImpl.java:818)
	at com.mysql.jdbc.ResultSetImpl.getStringInternal(ResultSetImpl.java:5651)
	at com.mysql.jdbc.ResultSetImpl.getString(ResultSetImpl.java:5570)
	at com.sapphire.etl.AERSDemoVerify.demoCaseGenderDEC(AERSDemoVerify.java:257)
	... 24 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m4">com.sapphire.etl.AERSDemoVerify#demoCaseValidTo</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where valid_to is null<br/>Expected : [0]<br/>Actual : [13]</div></td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: SELECT count(*) FROM aers_demo where valid_to is null
	at org.testng.Assert.fail(Assert.java:83)
	at com.sapphire.etl.AERSDemoVerify.demoCaseValidTo(AERSDemoVerify.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: Lists differ at element [0]: 0 != 13 expected [0] but found [13]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:549)
	at org.testng.Assert.assertEquals(Assert.java:514)
	at com.sapphire.etl.AERSDemoVerify.demoCaseValidTo(AERSDemoVerify.java:76)
	... 24 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m5">com.sapphire.etl.AERSDemoVerify#demoCASE_ID</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where case_id is null<br/>Expected : [0]<br/>Actual : [0]</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m6">com.sapphire.etl.AERSDemoVerify#demoCaseSeriousness</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where seriousness = 1 and ( outc_death= 0 and  outc_life_threat =0 and outc_hosp =0 and outc_disability=0 and outc_cong_anomaly=0  and outc_reqd_interv=0 and outc_other=0 and outc_none=0 and outc_rec=0 and outc_rx=0)<br/>Expected : [0]<br/>Actual : [0]</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m7">com.sapphire.etl.AERSDemoVerify#demoCaseValidFrom</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where valid_from is null<br/>Expected : [0]<br/>Actual : [0]</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m8">com.sapphire.etl.AERSDemoVerify#demoFDA_date</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where fda_date is null<br/>Expected : [0]<br/>Actual : [0]</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m9">com.sapphire.etl.AERSDemoVerify#demoISR_ID</h3><table class="result"><tr><th>Messages</th></tr><tr><td><div class="messages">SELECT count(*) FROM aers_demo where isr_id is null<br/>Expected : [0]<br/>Actual : [0]</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p></body></html>